### PR TITLE
Auto verify placements can be correctly created/displayed/deleted

### PR DESCRIPTION
**Pull request by zxiao-redhat**
Hi @Phaow, @kasturinarra, @xingxingxia, I added a test case for testing the basic operations for KCP placements. Please take a look if you have time. Thanks!
Test:
```
passed: (57.3s) 2022-11-29T02:13:52 "[area/transparent-multi-cluster] Author:zxiao-Critical-Verify placements can be correctly created/displayed/deleted for shared compute [Suite:kcp/smoke/parallel]"

1 pass, 0 skip (57.3s)
```

### CODE REVIEW

**Comment by kasturinarra** [comment]
@zimo-xiao may i know the use case that you are trying to solve here ? Thanks !!

File: test/extended/placements/placements.go [hunk 1]
```diff
@@ -113,6 +114,47 @@ var _ = g.Describe("[area/transparent-multi-cluster]", func() {
...
+		g.By("# Delete placement one-by-one while checking if all placements are correctly displayed")
+		for i, placementName := range placementNames {
+			g.By(">	#.1 Delete placement")
+			err = k.WithoutNamespace().WithoutKubeconf().Run("delete").Args("placement", placementName).Execute()
+			o.Expect(err).NotTo(o.HaveOccurred())
 
+			g.By(">	#.2 Check if all placements are correctly displayed")
+			output, err = k.WithoutNamespace().WithoutKubeconf().Run("get").Args("placement").Output()
+			o.Expect(err).NotTo(o.HaveOccurred())
+			o.Expect(output).NotTo(o.ContainSubstring(placementName))
```
Comment (Phaow) [inline test/extended/placements/placements.go]: There may has a flake that delete placement is asynchronous action , using `o.Eventually` check the deleted resource finally deleted successfully(get not display) is better.

File: test/extended/testdata/placement/placement.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  name: "$(NAME)"
+spec:
+  locationResource:
+    group: workload.kcp.dev
+    resource: synctargets
+    version: v1alpha1
+  locationSelectors:
+    - {}
+  locationWorkspace: "$(LOCATION_WORKSPACE)"
+  namespaceSelector: {}
```
Comment (Phaow) [inline test/extended/testdata/placement/placement.yaml]: We' better add a blank line end of the code file to eliminate the `Warning`.

File: test/extended/placements/placements.go [hunk 3]
```diff
@@ -113,6 +114,47 @@ var _ = g.Describe("[area/transparent-multi-cluster]", func() {
...
+			for j := i + 1; j < len(placementNames); j++ {
+				availablePlacementName := placementNames[j]
```
Comment (Phaow) [inline test/extended/placements/placements.go]: I saw we just basic get the placements not filter the status of the placements. Could we check the status/display field is as expected?

Comment (zxiao-redhat) [reply test/extended/placements/placements.go]: Hi @Phaow, status is not supported by the KCP command. I tried to query it using `-o=jsonpath`. Yet, the operation takes more than ten minutes to complete in a double loop. I don't think querying status right now is a viable solution given its complexity.

**Comment by zxiao-redhat** [comment]
Hi @kasturinarra, this is the case that I will be proposing after the automation. It's a basic task that touches all the common operations of placements.

**Comment by zxiao-redhat** [comment]
Hi @kasturinarra @Phaow @xingxingxia, I updated the case based on your comments. PTAL. Thanks!
```
passed: (33.7s) 2022-12-08T03:18:23 "[area/transparent-multi-cluster] Author:zxiao-Critical-Verify placements can be correctly created/displayed/deleted for shared compute [Suite:kcp/smoke/parallel]"

1 pass, 0 skip (33.7s)
```

**Comment by kasturinarra** [comment]
> Hi @kasturinarra, this is the case that I will be proposing after the automation. It's a basic task that touches all the common operations of placements.

@zimo-xiao I think we better not do that. We propose the test case first, get it reviewed and approved then go for automation. Also i have automated similar cases, i am just wondering why do we have to create things in loop ? This case i am not sure if suitable for functional/regression tests. And deletion part you could add to the existing cases it self may be ?

**Pull request closed without merging**